diff -ruN a/src/Ifsnop/Mysqldump/Mysqldump.php b/src/Ifsnop/Mysqldump/Mysqldump.php
--- a/src/Ifsnop/Mysqldump/Mysqldump.php
+++ b/src/Ifsnop/Mysqldump/Mysqldump.php
@@ -228,6 +228,16 @@
     }
 
     /**
+     * Get table column types.
+     *
+     * @return array
+     */
+    protected function tableColumnTypes()
+    {
+      return $this->tableColumnTypes;
+    }
+
+    /**
      * Keyed by table name, with the value as the conditions:
      * e.g. 'users' => 'date_registered > NOW() - INTERVAL 6 MONTH AND deleted=0'
      *
